modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DXPCloudSetup.testcase

```diff
@@ -27,514 +27,667 @@ definition {
+		}
 	}
 
-	@description = "LPS-144986 - Verify if Submit button is only enabled when all the fields are filled"
+	@description = "Automation ID: LPS-144986 | Test Summary: Verify if Submit button is only enabled when all the fields are filled"
 	@priority = "5"
 	test CanClickSubmitButtonWithAllFilledIn {
-		Accounts.openAccountsAdmin();
-
-		CustomerPortal.accountTableProject(title = "Test Account 01");
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
 
-		Accounts.addAssigneeToRoleViaUsersTab(
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
-
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		CustomerPortal.clickOnStartProjectSetup();
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.clickOnSkipForNowButton();
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CP2DXPCloudSetup.typeAllSetupDXPCloudInfoFields();
+		task ("Going through onboarding to Setup") {
+			CustomerPortal.onboardingFlow(cloudEnvironment = "true");
+		}
 
-		AssertElementPresent(locator1 = "CP2DXPCloudSetup#SUBMIT_BUTTON");
+		task ("Assert that the submit button is enabled") {
+			AssertElementPresent(locator1 = "CustomerPortalSite#SUBMIT_BUTTON");
+		}
 	}
 
-	@description = "LPS-144990 - If the admin user selects 'Skip for Now' then the user is taken to that project's home page / subscriptions page"
+	@description = "Automation ID: LPS-144990 | Test Summary: If the admin user selects 'Skip for Now' then the user is taken to that project's home page / subscriptions page"
 	@priority = "4"
 	test CanSkipForNowToOverview {
-		Accounts.openAccountsAdmin();
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
 
-		CustomerPortal.accountTableProject(title = "Test Account 01");
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		Accounts.addAssigneeToRoleViaUsersTab(
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.clickOnStartProjectSetup();
+		task ("Going through onboardig") {
```

Comment:
Luana, please check the typo: "onboarding"

Reply:
Done.